examples/functions/timetable/Netex_01.2_Bus_SimpleTimetable_WithTimings.xml

```diff
@@ -1291,6 +1293,16 @@ The Calendar is shown coded as
 					<FrameDefaults>
 						<DefaultCodespaceRef ref="acs"/>
 					</FrameDefaults>
+					<contracts>
+						<Contract version="1.0" id="acs:bus_services">
+							<contractees>
+								<AuthorityRef version="1.0" ref="acs:Transport_Sigma"/>
+							</contractees>
+							<contractors>
+								<OrganisationRef version="1.0" ref="acs:ACS"/>
```

Comment:
Check if OrganisationRef is the best reference or we want to have AuthorityRef here.